src/components/comments/CommentSection.js

```diff
@@ -20,8 +20,7 @@ a corresponding question or answer*/
   /* Only rendering the ones that should be visible at that current moment*/
   let visibleFollowups = allFollowups.slice(0, props.currentVisibleText);
   return (
-    <div className="comments-container content-container ">
-      
+    <div className="comments-container content-container " id={props.name}>
```

Comment:
Is the only function of the `props.name` prop to pass on this `id`? If so, I think it may be better to just manually set the `id` of the component when we make it, though I'm not sure what's best practice here.